### DISCUSSION TITLE
debug flag does not work in v0.57.0

### DISCUSSION THREAD

**Discussion by dave505** [aquasecurity/trivy-aws#256] [Q&A]
### Description

Using the --debug flag or -d results in an unknown flag error when used with trivy v0.57.0. The same command works in v0.52.2

### Desired Behavior

Debug output to appear

### Actual Behavior

2024-11-14T11:00:03Z	FATAL	unknown flag: --debug	<nil>

### Reproduction Steps

```bash
1.trivy aws --service s3 --debug --check-namespaces <some_namespace> --format json --output ./filename.json
```

### Target

AWS

### Scanner

Vulnerability

### Output Format

JSON

### Mode

Standalone

### Debug Output

```bash
2024-11-14T11:00:03Z	FATAL	unknown flag: --debug	<nil>
```

### Operating System

macOS Sonoma 14.5

### Version

```bash
Version: 0.57.0
```

### Checklist

- [X] Run `trivy clean --all`
- [X] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by DmitriyLewen** [comment]
Hello @dave505 

`aws` command has been removed from Trivy.
You can scan `aws` using plugin.

More info in aquasecurity/trivy#6818

Regards, Dmitriy

**Reply by dave505** [reply]
Hi Dmitry,

I installed the aws plugin with `trivy plugin install github.com/aquasecurity/trivy-aws` before I tried to use the --debug flag.

Otherwise you receive the message `Trivy AWS is now available as an optional plugin. See github.com/aquasecurity/trivy-aws for details.`

**Comment by pazarr** [comment]
I was able to reproduce the issue on MacOS Sonoma and trivy version 0.54.1 with the aws plugin v0.14.02.
```
$ trivy aws --debug
2024-11-14T14:13:38+01:00	FATAL	unknown flag: --debug	<nil>
```

**Comment by nikpivkin** [comment]
This was fixed in v0.14.2, but Trivy loads v0.14.1, which is specified in the plugin manifest. I opened a [PR](https://github.com/aquasecurity/trivy-aws/pull/255) to update the manifest. We will improve our release process to avoid such problems in the future.

**Comment by nikpivkin** [comment]
Moved this discussion to the `trivy-aws` repository, as this is a plugin issue.

**Comment by nikpivkin** [comment]
Track https://github.com/aquasecurity/trivy-aws/issues/254
